buildSrc/src/main/kotlin/LocalProtoktBuild.kt

```diff
@@ -23,9 +23,7 @@ import protokt.v1.gradle.configureProtokt
 import java.io.File
 
 fun Project.localProtokt(disableJava: Boolean = true) {
-    configureProtokt(this, null, disableJava) {
-        "$rootDir/protokt-codegen/build/install/$CODEGEN_NAME/bin/$CODEGEN_NAME"
-    }
+    configureProtokt(this, null, disableJava, "$rootDir/protokt-codegen/build/install/$CODEGEN_NAME/bin/$CODEGEN_NAME")
```

Comment:
No need for deferred evaluation anymore and this was causing a problem in the plugin integration test as it tried to create the new `_codegen_` configuration once per target.